client/api/system/v1alpha1/api.pb.go

```diff
@@ -0,0 +1,303 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.25.0-devel
+// 	protoc        v3.11.2
+// source: src/github.com/kubernetes-csi/csi-proxy/client/api/system/v1alpha1/api.proto
+
+package v1alpha1
+
+import (
+	context "context"
+	proto "github.com/golang/protobuf/proto"
+	grpc "google.golang.org/grpc"
+	codes "google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+// This is a compile-time assertion that a sufficiently up-to-date version
+// of the legacy proto package is being used.
+const _ = proto.ProtoPackageIsVersion4
```

Comment:
Can you please do `go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.2` to make sure this continues to use the (slightly) older version of `protoc-gen-go` like the rest of the API groups like [this](https://github.com/kubernetes-csi/csi-proxy/blob/master/client/api/disk/v1beta1/api.pb.go#L25)? I would like to understand the consequences of bumping the version of individual API groups on clients a little more before ideally bumping it for all the API groups. If a client plugin is able to handle both v1.4 and v1.3 simultaneously, then I have no concerns.